package.json

```diff
@@ -29,6 +29,7 @@
     "@angular/platform-browser": "~8.2.14",
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
+    "dexie": "^3.0.3",
```

Comment:
We still rename "main.js" -> shouldn't we stop doing that?

Reply:
Yes, I will remove it.